resources/resource_ids.spec

```diff
@@ -95,6 +95,9 @@
   "brave/components/brave_ads/resources/bat_ads_resources.grd": {
     "includes": [51200]
   },
+  "brave/components/brave_federated/resources/static_resources.grd": {
+    "includes": [52230]
```

Comment:
Please include a `"META": {"sizes": {"includes": [1]}},`

Reply:
Out of curiosity, what is the difference to the ones not including the "META" field?